tests/services/test_savedsearch.py

```diff
@@ -1,15 +1,22 @@
+import pytest
 from tests.conftest import create_mock_response
 
 from py42 import settings
+from py42.services._connection import Connection
 from py42.services.fileevent import FileEventService
 from py42.services.savedsearch import SavedSearchService
 
 SAVED_SEARCH_GET_RESPONSE = """
     {"searches": [{"groups": [] }]}
 """
+FILE_EVENT_URI = "/forensic-search/queryservice/api/v1/fileevent"
 
 
 class TestSavedSearchService:
+    @pytest.fixture
+    def connection(self, mocker):
```

Comment:
I don't think this is needed.

The `mock_connection` fixture is already being imported here, and it is basically the same thing.